### ISSUE TITLE
Item List in 'int main'

### ISSUE THREAD

**Issue by KJKipuw** [ETE-1151-F24/project-2-group-b-random-group-2#2]
Can we move this item list outside of 'int main'? 

https://github.com/ETE-1151-F24/project-2-group-b-random-group-2/blob/c0bc75f2954990ddc81bfe197edcb1bf91bf704e/project%201.cpp#L95

**Comment by hermannkenko** [comment]
Yes sure! I am working on it right now. This is called modularity.
